rdprail-shell/shell.c

```diff
@@ -3420,8 +3391,7 @@ activate(struct desktop_shell *shell, struct weston_view *view,
 
 	if (shell->rdprail_api->notify_window_zorder_change)
 		shell->rdprail_api->notify_window_zorder_change(
-			shell->compositor,
-			shell->focus_proxy_surface == es ? NULL : es);

```

Comment:
When focus is lost, here still need let backend know (by calling with NULL), so backend will send Z-order update message with marker window (aka Windows local window) at foreground.